README.md

```diff
@@ -123,3 +123,13 @@ module.exports = {
 ```
 
 We also provide `@cybozu/eslint-config/presets/kintone-customize-es5-prettier` to use it with `prettier`.
+
+## ⚠️ Class JSX Syntax
```

Comment:
@nissy-dev
Thank you! I have an additional request 🙏 
I want to add the `## React Support` section and move this to the upper of the `## For kintone customize developers` section.

```suggestion
## React Support

### ⚠️ Classic JSX Syntax
```

Could you apply the change and move the section to the upper of the `## For kintone customize developers` section?

Reply:
@koba04 
I fixed!